### PR TITLE
fix: Add dump process data function

### PR DESCRIPTION
**Pull request by filipecabaco**
## What kind of change does this PR introduce?

To help debugging potential issues with processes being created without restrictions were adding a function to help debug those issues.

### CODE REVIEW

**Review by abc3 — APPROVED** [review]

File: lib/realtime/operations.ex [hunk 1]
```diff
@@ -98,4 +98,58 @@ defmodule Realtime.Operations do
       # credo:disable-for-next-line
       IO.inspect({"Failed to stop", tenant, kind, reason})
   end
+
+  def dump_processes do
+    {:ok, device} = File.open("/tmp/erlang_process_dump.csv", [:write, :utf8])
+
+    device
+    |> IO.write(
+      "intial_call\tstatus\tmemory\tstack_size\tcurrent_stacktrace\tmessage_queue_len\tmessages\n"
```
Comment (chasers) [inline lib/realtime/operations.ex:107]: You can do `term_to_binary` and just write the whole term from `Process.list()`

**Review by chasers — CHANGES_REQUESTED** [review]
I'd also like to see an `analyze` fn which lets you give it a process list and groups processes by something in the dictionary and gives you counts of each group.

File: lib/realtime/operations.ex (hunk 1 above)
Comment (filipecabaco) [reply lib/realtime/operations.ex:107]: 👍 will change

**Review by chasers — APPROVED** [review]

**Pull request merged**